libs/components/src/lib/components.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+/* eslint-disable-next-line */
+export interface ComponentsProps {}
+
+const StyledComponents = styled.div`
+  color: pink;
+`;
+
+export function Components(props: ComponentsProps) {
+  return (
+    <StyledComponents>
+      <h1>Welcome to Components!</h1>
```

Comment:
or delete text or i18n, no text should be in components

Reply:
deleted